egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh

```diff
@@ -16,8 +16,12 @@ decode_dir_affix=
 num_epochs=4
 remove_egs=false
 common_egs_dir=
-
 affix=
+
+# decoding options
+extra_left_context=50
```

Comment:
Problem is that there are no $chunk_{left,right}_context variables 
anymore. I remove the options and use the value 40 as Vimal suggested.

在 2016/4/21 5:23, Daniel Povey 写道:
>
> In egs/fisher_swbd/s5/local/chain/run_blstm_6h.sh 
> <https://github.com/kaldi-asr/kaldi/pull/657#discussion_r60490302>:
>
> >  affix=
> > +
> > +# decoding options
> > +extra_left_context=50
>
> I assume you mean chunk_left_context-- I'd say better to do
>
> |extra_left_context=$[$chunk_left_context+10] |
>
> .. no conditionals as not really needed, and $[] is used in most of 
> the scripts for basic math like this.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/657/files/f9483fdcb59dbfa711df31c41c2bea90b23c142c..ea76204bc64d9a139597e488a7821d5e44dc3bd4#r60490302>
>